### ISSUE TITLE
Customize Colors of the Generated QR Code

### ISSUE THREAD

**Issue by saifkhichi96** [journeyapps/zxing-android-embedded#698]
#### Description of the problem:

Quoting the [README](https://github.com/journeyapps/zxing-android-embedded/blob/master/README.md), 

> No customization of the image is currently supported, including changing colors or padding. If you
require more customization, copy and modify the source for the encoder.

The QR codes are generated in black color with a white background by default. The colors are hardcoded in the [BarcodeEncoder.java](https://github.com/journeyapps/zxing-android-embedded/blob/master/zxing-android-embedded/src/com/journeyapps/barcodescanner/BarcodeEncoder.java) class.

```
private static final int WHITE = 0xFFFFFFFF;
private static final int BLACK = 0xFF000000;

public Bitmap createBitmap(BitMatrix matrix) {
    ...
    for (int y = 0; y < height; y++) {
        ...
        for (int x = 0; x < width; x++) {
            pixels[offset + x] = matrix.get(x, y) ? BLACK : WHITE;
        }
    }
    ...
    return bitmap;
}
```

Color customization is a common requirement of most end-users of the library, who might want to place the QR code on a non-white background. And it is very easy to provide this functionality by just letting the WHITE and BLACK variables be non-static and providing setter methods for setting a [@ColorInt](https://developer.android.com/reference/androidx/annotation/ColorInt) value for both of them. This would allow the users to set the background and foreground colors of their own choice for the generated codes, and even make the background transparent if they need to.

It would be useful to have this feature out-of-the-box in the library itself, instead of users needing to modify the source code themselves.

**Which library version are you using?** 4.3.0

**Comment by saifkhichi96** [comment]
Fixed in #696

**Comment by erdzan12** [comment]
As this has been fixes is it possible to color the edges of the QR code like described in this issue?
https://github.com/journeyapps/zxing-android-embedded/issues/364
